test/e2e_node/cpu_manager_test.go

```diff
@@ -280,31 +262,32 @@ func configureCPUManagerInKubelet(f *framework.Framework, policyName string, cle
 	return oldCfg
 }
 
-func runGuPodTest(f *framework.Framework, cpuCount int) {
-	var pod *v1.Pod
+func getAllowedCPUs(f *framework.Framework, namespace, podName, cntName string) cpuset.CPUSet {
+	logs, err := e2epod.GetPodLogs(f.ClientSet, namespace, podName, cntName)
+	framework.ExpectNoErrorWithOffset(1, err, "expected log not found in container [%s] of pod [%s]", cntName, podName)
 
-	ctnAttrs := []ctnAttribute{
+	framework.Logf("got pod logs: %v", logs)
+	cpus, err := cpuset.Parse(strings.TrimSpace(logs))
+	framework.ExpectNoErrorWithOffset(1, err, "parsing cpuset from logs for [%s] of pod [%s]", cntName, podName)
```

Comment:
we should stop using `WithOffset`, just add `ginkgo.GinkgoHelper()` at the beginning of the helper functions.